### PR TITLE
Optimization: Precompute AStar between waypoints

### PR DESCRIPTION
**Pull request by mads256h**

### CODE REVIEW

File: Assets/Scripts/ExperimentSimulations/ConscientiousReactiveExperiment.cs [hunk 1]
```diff
@@ -127,7 +127,8 @@ private void Start()
                     { "conscientious_reactive", _ => new ConscientiousReactiveAlgorithm() },
                 };
             var buildingMaps = ((buildingConfigList100));
-            foreach (var mapConfig in buildingMaps)
+            var caveMaps = new List<CaveMapConfig>() { new CaveMapConfig(randNumbers[0]) };
+            foreach (var mapConfig in caveMaps)
             {
```
Comment (S3JER) [inline Assets/Scripts/ExperimentSimulations/ConscientiousReactiveExperiment.cs:132]: Skal dette med ind?

File: Assets/Scripts/Map/Visualization/Patrolling/NoneVisualizationMode.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+// or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+// Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License along
+// with MAES. If not, see http://www.gnu.org/licenses/.
+// 
+// Contributors: Malte Z. Andreasen, Philip I. Holler and Magnus K. Jensen
+// 
+// Original repository: https://github.com/MalteZA/MAES
```
Comment (S3JER) [inline Assets/Scripts/Map/Visualization/Patrolling/NoneVisualizationMode.cs:20]: Remove?

**Review by S3JER — CHANGES_REQUESTED** [review]
Tjek lige de to kommentar

File: Assets/Scripts/Map/Visualization/Patrolling/NoneVisualizationMode.cs (hunk 2 above)
Comment (mads256h) [reply Assets/Scripts/Map/Visualization/Patrolling/NoneVisualizationMode.cs:20]: No it good

File: Assets/Scripts/ExperimentSimulations/ConscientiousReactiveExperiment.cs (hunk 1 above)
Comment (mads256h) [reply Assets/Scripts/ExperimentSimulations/ConscientiousReactiveExperiment.cs]: fixed

**Review by S3JER — APPROVED** [review]

**Pull request merged**